### ISSUE TITLE
SCSS Variable

### ISSUE THREAD

**Issue by naher94** [DeMoorJasper/vscode-pigments#14]
Would be awesome if there was a way for it display the color of SCSS variables. Thanks for the extension!

![Screen Shot 2021-04-24 at 7 05 25 PM](https://user-images.githubusercontent.com/7800629/115975121-083a8c00-a530-11eb-97c1-bac06454cbb4.png)

**Comment by DeMoorJasper** [comment]
Yeah that would be really cool, will put it on my todo for whenever I have time. Should definitely be doable, just a couple new rules to add.

If anybody else wants to look into this before that feel free to do so, would love to help

**Comment by naher94** [comment]
Wanted to follow up and if this would still be on the table to add?
